x-pack/test/security_solution_endpoint/apps/integrations_feature_flag/mocks.ts

```diff
@@ -0,0 +1,807 @@
+/*
```

Comment:
Copied this file from ../integrations. It is used with FF specific FTRs and once this feature is GA this file should be copied over to ../integrations.